Jenkinsfile

```diff
@@ -3,7 +3,7 @@
 
 pipeline {
   agent {
-    label 'docker && linux'
+    label 'linux-amd64-docker'
```

Comment:
```suggestion
    // 'docker' is the (legacy) label used on ci.jenkins.io for "Docker Linux AMD64" while 'linux-amd64-docker' is the label used on infra.ci.jenkins.io
    label 'docker || linux-amd64-docker'
```

Reply:
Isn't there a risk of getting a Windows Docker agent with `docker` on ci.jenkins.io?

Reply:
Good question! But the answer is no:
- Azure Windows VM labels on ci.jenkins.io: https://github.com/jenkins-infra/jenkins-infra/blob/production/hieradata/clients/azure.ci.jenkins.io.yaml#L346
- EC2 Windows VM labels on ci.jenkins.io: https://github.com/jenkins-infra/jenkins-infra/blob/production/hieradata/clients/azure.ci.jenkins.io.yaml#L428


=> we still have to update the label policy to a "safe" on on ci.jenkins.io, but right now, `docker` is safe (thanks to your work as far as I remember)